astropy/modeling/optimizers.py

```diff
@@ -216,11 +216,10 @@ def __call__(self, objfunc, initval, fargs, **kwargs):
         """
         if 'maxiter' not in kwargs:
             kwargs['maxiter'] = self._maxiter
-        if 'acc' not in kwargs:
-            kwargs['acc'] = self._acc
+  
```

Comment:
Yep, these are the two options and I went with the first one. I can see good and bad things either way. But I like the way you implemented it now, since both parameters work so :+1: for merging after tests pass.